CHANGES.md

```diff
@@ -1,7 +1,9 @@
 # Release-Changelog
 
-## 1.4.5 (TBD)
+## 1.5.0 (TBD)
```

Comment:
The changes should be added under a second called 'TBD'. I do not know if we had it back then, but if you rebase your branch onto the current develop, you should see a section with 'TBD'.

Reply:
fixed